hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import styles from "./CartCard.module.scss";
+import Typography from "@material-ui/core/Typography";
+import Card from "@material-ui/core/Card";
+import CardMedia from "@material-ui/core/CardMedia";
+import CardContent from "@material-ui/core/CardContent";
+import CardActions from "@material-ui/core/CardActions";
+import Select from "@material-ui/core/Select";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
+
+const Selections = (currentStock) => {
+    const items = [];
+    let i = 0;
+
+    for (i = 1; i < currentStock + 1; i++) {
+        items.push(
+            <MenuItem key={i} role="quantity" value={i}>
+                {i}
+            </MenuItem>
+        );
+    }
+    return items;
+};
+
+const Stock = ({ currentStock, handleChange, item }) => {
```

Comment:
`item` isn't the best prop name. Can you use something more descriptive of what it is?